compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaLayer.skiko.kt

```diff
@@ -179,11 +180,15 @@ internal class SkiaLayer(
             rotateX(rotationX)
             scale(scaleX, scaleY)
         }
-        matrix *= Matrix().apply {
-            // the camera location is passed in inches, set in pt
-            val depth = cameraDistance * 72f
-            set(row = 2, column = 3, v = -1f / depth)
-            set(row = 2, column = 2, v = 0f)
```

Comment:
Do you know why was it set to 0 before?
> `set(row = 2, column = 2, v = 0f)`

Reply:
To get correct/matches Android result. And visually (rendering transformation) it worked great